### DISCUSSION TITLE
Code editor is disabled?

### DISCUSSION THREAD

**Discussion by shhlife** [nxt-dev/nxt_editor#133] [Q&A] [answered]
I installed nxt using a virtual environment as suggested, launched the editor, and created a node. However, the code editor appears disabled. I can't seem to type into it. Is there a trick to enable editing?

thanks!
-Jason

![image](https://user-images.githubusercontent.com/5397345/104414958-f48d8b80-5525-11eb-8f97-54e7f8b780f5.png)

**Comment by shhlife** [comment, marked as answer]
Oy - newbie problem. :) Just double-click in the editor and that makes it editable. Not sure how I didn't figure that out right away - but maybe something to think about that it's not super clear for new users that double-clicking allows you to edit.

**Reply by ImLucasBrown** [reply]
Ah, you're not the first to have this confusion.
We've talked at length about whether or not it should be a double click to enter editing. Ultimately its a protection against accidentally localizing or editing code.
Maybe we can provide some visual feedback if a user tries to type when the editor is locked. 🤔

**Reply by shhlife** [reply]
I think that's a great idea - a tooltip or something that says "Hey! Totally get that you want to edit your code - try double-clicking! :)"

**Comment by MichaelAldrich** [comment]
Good news! In the latest editor release(3.7.0) there is a "Double click to edit" message that displays over the code editor.

![gif](https://cdn.discordapp.com/attachments/781220843568103474/806897342270668830/CodeEditorMsg01.gif)
